src/auth/device-auth-flow.ts

```diff
@@ -207,6 +207,32 @@ export async function refreshAccessToken(selectedScopes?: string[]): Promise<str
   }
 }
 
+export async function revokeRefreshToken() {
```

Comment:
There are no tests for this new functionality tests for both the revocation function and logout integration. I think we can at least mock out some the actual calls to get some test coverage.